### ISSUE TITLE
Allow PropertySortOrder to ignore subproperties order

### ISSUE THREAD

**Issue by Anahkiasen** [brigade/scss-lint#210]
Currently because of the way the properties order linter works, it's _very_ specific about the order subproperties come in. Per example this doesn't pass:

``` css
body {
    background-image: image-url('backgrounds/package.png');
    background-position: 0 7em;
    background-repeat: no-repeat;
}
```

Only because repeat should, for concentric, come before position. While a global sort order is very valuable, there is no real logic in the way the subproperties were ordered (excepted the left/right/top/bottom ones). Could a flag be added to ignore those ? Or is that not technically possible.
